src/Microsoft.Health.Dicom.Web/appsettings.Development.json

```diff
@@ -15,7 +15,9 @@
   },
   "DicomServer": {
     "Features": {
-      "EnableUpsRs": true
+      "EnableUpsRs": true,
+      "EnableDualWrite": false,
+      "SupportNewBlobFormatForNewService": true
```

Comment:
This also seems problematic if you pull latest and hit F5

Reply:
Yeah, I know I would break everyone. I thought of informing the team to clean up the environment.